lib/plans/checkout.dart

```diff
@@ -149,6 +148,14 @@ class _CheckoutState extends State<Checkout>
     return widgets;
   }
 
+  Future<void> checkProUser() async {
+    final res = await ffiProUser();
+    if (!widget.isPro && res == "true") {
```

Comment:
@atavism  Here I think it should be ` if (!widget.isPro && res.toDartString() == "true")`